### PR TITLE
[muon id] added a few more working points

### PR DESCRIPTION
**Pull request by drkovalskyi**
For details see:
- https://its.cern.ch/jira/browse/CMSMUONS-95
- https://its.cern.ch/jira/browse/CMSMUONS-97
- MultiIso was requested by SUSY PAG

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
@drkovalskyi : please allow somehow "muon" or "muon ID" in the title, so that the topic of the PR can be easily understood from it

File: DataFormats/MuonReco/src/MuonSelectors.cc [hunk 1]
```diff
@@ -1004,6 +1022,7 @@ void muon::setCutBasedSelectorFlags(reco::Muon& muon,
   if (dbCorectedRelIso<0.20)    selectors |= reco::Muon::PFIsoMedium;
   if (dbCorectedRelIso<0.15)    selectors |= reco::Muon::PFIsoTight;
   if (dbCorectedRelIso<0.10)    selectors |= reco::Muon::PFIsoVeryTight;
+  if (dbCorectedRelIso<0.05)    selectors |= reco::Muon::PFIsoVeryVeryTight;
```
Comment (perrotta) [inline DataFormats/MuonReco/src/MuonSelectors.cc]: The point is ultra-minor... However, I let you decide if you want to fix the spelling of the name of this variable ("Corected" -> "Corrected") or not. Just either fix it or let us know that you don't plan doing it.
(I have no other issues with this PR)

**Comment by perrotta** [comment]
@drkovalskyi : by the way, do you plan to backport this PR to 94X for the re-miniAOD?

**Comment by perrotta** [comment]
please test
(just to re-create the jenkins outputs, which have disappeared in the meanwhile)

**Comment by drkovalskyi** [comment]
I will correct typos. We don't plan to do any back ports unless changes 94X will be used for full Run2 re-reco.

**Comment by perrotta** [comment]
@drkovalskyi : test results are out, and as expected the only changes are the extra bits for the additional selectors in reco and miniAOD outputs

Please let me know if you plan to fix the spelling of the local variable, as you anticipate in https://github.com/cms-sw/cmssw/pull/22420#issuecomment-372593687: in that case I will wait to sign it until that update arrives

**Comment by drkovalskyi** [comment]
I will correct the typos - I just need to find some quiet time to do it.

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Four extra bits added for the additional selectors in reco and miniAOD outputs
- Changes in jenkins outputs just reflect it

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Pull request merged**